### PR TITLE
Adding span count attribute

### PR DESCRIPTION
**Pull request by sarthak77**
Ref issue : hypertrace/hypertrace-ui#1157

### CODE REVIEW

**Review by kotharironak — APPROVED** [review]

File: pinot-servicemanager/schemas/spanEventView-schemaFile.json [hunk 1]
```diff
@@ -148,6 +148,11 @@
       "dataType": "STRING",
       "singleValueField": false,
       "defaultNullValue": ""
+    },
+    {
+      "name": "span_count",
```
Comment (kotharironak) [inline pinot-servicemanager/schemas/spanEventView-schemaFile.json:153]: `span_count` -> `spans`?

Comment (kotharironak) [reply pinot-servicemanager/schemas/spanEventView-schemaFile.json:153]: Just show that query-service side, you have mapped to `spans`. So, pl. rename this column to `spans`.

**Review by kotharironak — CHANGES_REQUESTED** [review]

File: pinot-servicemanager/schemas/spanEventView-schemaFile.json (hunk 1 above)
Comment (sarthak77) [reply pinot-servicemanager/schemas/spanEventView-schemaFile.json]: done

**Review by kotharironak — DISMISSED** [review]

File: pinot-servicemanager/schemas/spanEventView-schemaFile.json [hunk 2]
```diff
@@ -148,6 +148,11 @@
       "dataType": "STRING",
       "singleValueField": false,
       "defaultNullValue": ""
+    },
+    {
+      "name": "spans",
+      "dataType": "INT",
+      "defaultNullValue": 0
```
Comment (aaron-steinfeld) [inline pinot-servicemanager/schemas/spanEventView-schemaFile.json:155]: Default value should be 1 for all historical rows

**Review by aaron-steinfeld — CHANGES_REQUESTED** [review]

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**